L1Trigger/Phase2L1ParticleFlow/src/egamma/pftkegalgo_ref.cpp

```diff
@@ -115,16 +129,44 @@ edm::ParameterSetDescription l1ct::PFTkEGAlgoEmuConfig::IsoParameters::getParame
   return description;
 }
 
-l1ct::PFTkEGAlgoEmuConfig::CompIDParameters::CompIDParameters(const edm::ParameterSet &pset)
-    : CompIDParameters(pset.getParameter<double>("loose_wp"),
-                       pset.getParameter<double>("tight_wp"),
-                       pset.getParameter<std::string>("model")) {}
+l1ct::PFTkEGAlgoEmuConfig::CompIDParameters::CompIDParameters(const id_score_t &bdtScore_loose_wp, const id_score_t &bdtScore_tight_wp, const std::string &model)
```

Comment:
Add a constructor from vectors and "simple" tupes to be used in vitis TB

Reply:
Now there is a new constructor to use together with the createWP factory method

```cpp
auto comp_params = CompIDParameters(createWP(-999.), createWP("pt", {0,20,50}, {0.1,0.2,0.3}), "model.json");
```